guides/working-remotely.md

```diff
@@ -0,0 +1,25 @@
+# Working remotely
+
+When a delivery team aren't all in the same place, it's important that we continue to communicate and collaborate as well as we can. This guide contains some suggestions about how to create a happy and productive remote working environment.
+
+## Coming and going
+
+When you arrive, go home, or leave your station for a while, let your teammates know in Slack.
+
+## Know when you're having a meeting
+
+It is easy for a casual conversation around the office to turn into an ad-hoc team meeting. Recognise when this happens, and stop to include remote team members if possible. If they can't be present, youneed to give them a written note of the conversation.
```

Comment:
It's a lovely thing to have this written down. I had reconciled myself to catching up later through different channels being the norm.

There's a missing space in "you need".

Reply:
It feels like this more naturally belongs below "Keep Notes" below, largely because you've set out "short and sharp" there and it would be good to have this as the latest thing in mind when reading this section. Otherwise "you need to give them a written note" could be seen as extra effort and extra-curricular, when in fact it's just pointing them to the short, sharp thing you're already writing as a matter of course.